objects/screen.c

```diff
@@ -241,6 +243,143 @@ screens_exist(void)
     return globalconf.screens.len > 0;
 }
 
+#ifdef XCB_RANDR_GET_MONITORS
```

Comment:
Ok, but at least do `#ifdef XCB_RANDR_GET_MONITORS /* Monitors were introduced in RandR 1.5 */`, so that one doesn't have to figure this out from just `screen_scan_randr()`.